Userland/Applications/Presenter/SlideObject.cpp

```diff
@@ -82,6 +82,16 @@ Text::Text()
     REGISTER_STRING_PROPERTY("font", font, set_font);
 }
 
+Text::Text(Gfx::Color default_color)
+{
+    REGISTER_STRING_PROPERTY("text", text, set_text);
+    REGISTER_FONT_WEIGHT_PROPERTY("font-weight", font_weight, set_font_weight);
+    REGISTER_TEXT_ALIGNMENT_PROPERTY("text-alignment", text_alignment, set_text_alignment);
+    REGISTER_INT_PROPERTY("font-size", font_size, set_font_size);
+    REGISTER_STRING_PROPERTY("font", font, set_font);
+    this->set_color(default_color);
+}
+
```

Comment:
You can remove the no-argument constructor of Text because the SlideObject parser function should be the only user of the SlideObject (and children) constructors.